### ISSUE TITLE
[EMBEDDED] TMP336Z

### ISSUE THREAD

**Issue by Mogball** [teamwaterloop/GooseProject#2]
# TMP336Z
Code to pull values from contact temperature sensor.

**Comment by quintonlucyk** [comment]
Got it working on Josie's Uno, but couldn't get it working on team's Mega 2560

**Comment by Mogball** [comment]
Yeah there might be issues with that mega. If it works on Josie's board it'll probably work on a not messed up mega
